<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pbEdit">
       <property name="text">
        <string>Редактировать</string>
       </property>
       <property name="shortcut">
        <string>F2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pdAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="shortcut">
        <string>Ins</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pdDel">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbEdit</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>editRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pdAdd</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>addRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pdDel</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>delRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Settings</receiver>
   <slot>enableActionButtons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addRow()</slot>
  <slot>editRow()</slot>
  <slot>delRow()</slot>
  <slot>enableActionButtons()</slot>
 </slots>
</ui>
